test/build/50_ch-image.bats

```diff
@@ -329,3 +329,13 @@ EOF
 }
 EOF
 }
+
+@test 'ch-image push: consitent layer hash' {
+ run ch-image push --hash-only 00_tiny
+ [[ $status -eq 0 ]]
+ echo "$output"
+ [[ $output = *'gathering'*'preparing'*'metadata'*'layer 1/1'* ]]
+ out=$(ch-image push --hash-only 00_tiny)
+ out2=$(ch-image push --hash-only 00_tiny)
```

Comment:
I recommend a short pause here (2–3 seconds?) to make sure the timestamp increments.